### PR TITLE
Updates avant garde font

### PR DESCRIPTION
**Pull request by sarahscott**
simulator screen shot oct 9 2017 10 33 46 am

Skip New Tests
This depends on https://github.com/artsy/Artsy-OSSUIFonts/pull/14.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtworkRails/__tests__/ArtworkRailHeader-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/RelatedArtist-tests.tsx`
- `src/lib/Components/Text/__tests__/Headline-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Example/Emission/ARRootViewController.m [hunk 1]
```diff
@@ -135,7 +135,7 @@ - (ARCellData *)jumpToUserDocs
 - (ARCellData *)jumpToArtist
 {
   return [self tappableCellDataWithTitle:@"Artist" selection: ^{
-    id viewController = [[ARArtistComponentViewController alloc] initWithArtistID:@"david-shrigley"];
+    id viewController = [[ARArtistComponentViewController alloc] initWithArtistID:@"marina-abramovic-1"];
```
Comment (alloy) [inline Example/Emission/ARRootViewController.m]: YASS! Finally :tada:

File: src/data/fonts.ts [hunk 2]
```diff
@@ -4,5 +4,5 @@
 export default {
   "garamond-regular": "AGaramondPro-Regular",
   "garamond-italic": "AGaramondPro-Italic",
-  "avant-garde-regular": "Avant Garde Gothic ITCW01Dm",
+  "avant-garde-regular": "AvantGardeGothicITC",
```
Comment (alloy) [inline src/data/fonts.ts]: Will this work with the OSS fonts too? Or should this file return a different name for OSS users?

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtworkRails/__tests__/ArtworkRailHeader-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/RelatedArtist-tests.tsx`
- `src/lib/Components/Text/__tests__/Headline-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by mennenia** [comment]
Lookin' gooood! ✨ 

Changelog / CI 🍏 , then merge? :)

File: src/data/fonts.ts (hunk 2 above)
Comment (sarahscott) [reply src/data/fonts.ts]: It should be fine because I manually changed the metadata of the OSS avant garde to match it.

Comment (alloy) [reply src/data/fonts.ts]: Ah yes, I see 👍

Comment (sarahscott) [reply src/data/fonts.ts]: I'll try it out in OSS mode just to be sure though.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Home/ArtworkRails/__tests__/ArtworkRailHeader-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/RelatedArtist-tests.tsx`
- `src/lib/Components/Text/__tests__/Headline-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by sarahscott** [comment]
Merging now that it's 🍏

**Comment by alloy** [comment]
Sweeeeeet

**Comment by mennenia** [comment]
💃

**Pull request merged**